### DISCUSSION TITLE
Accept Bitcoin donations

### DISCUSSION THREAD

**Discussion by sovenor** [iterate-ch/cyberduck#15938] [Q&A]
Bitcoin donations are currently not accepted on the website. I would be happy to donate if Bitcoin was an option.

Some ways to add Bitcoin payments:

- Add a static Bitcoin address to the donation page using any Bitcoin wallet
- Implement [BTC Pay Server](https://btcpayserver.org/) to generate new addresses for each donation + enable new features

**Comment by tatze96** [comment]
We don't plan to add Bitcoin as a payment method.

**Reply by sovenor** [reply]
Ok thanks for the reply. Do you mind elaborating on why? @tatze96

**Comment by mar101xy** [comment]
i wanted to buy a license but i only use Bitcoin.
